Examcc/src/exo2/SortedContainerTest.java

```diff
@@ -0,0 +1,31 @@
+package exo2;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class SortedContainerTest {
+
+	@Test
+	void testsort() {
+		int [] list = {3,2,6,5,4,9,7,8,1};
+		int [] res = {1,2,3,4,5,6,7,8,9};
+		SortedContainer C = new SortedContainer(list);
+		C.sort(list, 8);
```

Comment:
Two comments:
- Should be done inside constructor...
- Why 8 when there are 9 elements?